src/actions/config.js

```diff
@@ -109,18 +117,16 @@ export function loadConfig() {
     try {
       const preloadedConfig = getState().config;
       const response = await fetch('config.yml', { credentials: 'same-origin' })
-      const requestSuccess = response.status === 200;
+      const fetchedConfig = await getConfigFromResponse(response);
 
-      if (!preloadedConfig && !requestSuccess) {
-        throw new Error(`Failed to load config.yml (${ response.status })`);
+      if (!preloadedConfig && !fetchedConfig) {
+        throw new Error(`Failed to load configuration.`);
```

Comment:
The change in error message is to open up to the config no longer being "config.yml" with the understanding that a failed request will already generate a message in the console.